org.eclipse.m2e.core.ui/src/org/eclipse/m2e/core/ui/internal/wizards/MavenDiscoveryProposalWizard.java

```diff
@@ -202,7 +202,7 @@ private void ignoreWorkspace(MojoExecutionKey key) {
         }
       };
 
-      Job job = new WorkspaceJob("Apply Lifecycle Mapping Changes") {
+      Job job = new WorkspaceJob("Apply Lifecycle Mapping Changes") { //$NON-NLS-1$
```

Comment:
This should typically be internationalized; hence `//$NON-NLS-1$` is erroneous here.